pkg/cvo/status.go

```diff
@@ -159,8 +159,9 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 	klog.V(5).Infof("Synchronizing errs=%#v status=%#v", validationErrs, status)
 
 	cvUpdated := false
+	availableUpdates := optr.getAvailableUpdates()
 	// update the config with the latest available updates
-	if updated := optr.getAvailableUpdates().NeedsUpdate(config); updated != nil {
+	if updated := availableUpdates.NeedsUpdate(config); updated != nil {
```

Comment:
Does `availableUpdates` need to kept around?

Reply:
oops, need to drop this hunk with the most-recent reroll.

Reply:
dropped the hunk with 0a0483a -> cd2d701.